### PR TITLE
Docs: Add Preact code snippet for Whats a Story

### PR DESCRIPTION
**Pull request by fbredius**
## What I did

I added a Preact code snipped for the `Get Started` -> `What's a Story` page. It is my first ever open source PR, so please be kind 😄. Once this is merged, I will add more code snippets for Preact.

## How to test

- Please read the files to check whether I adhere to the contributing guidelines.

Feedback is always welcome!

### CODE REVIEW

File: docs/snippets/preact/button-story.js.mdx
Comment (jonniebigodes) [inline docs/snippets/preact/button-story.js.mdx:11]: @fbredius if you could update the snippet and only use 2 spaces instead of 4 we'd appreciate it:
With that in mind the default export would be as such:
```js
export default {
  title: 'Button',
  component: Button,
}
```

This is basically to remain consistent, as some environments/setups might be configured for 4 spaces

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@fbredius thanks for the time and effort you've put into this pull request and your willingness to help us improve our documentation and include Preact snippets. 

Don't worry about anything me and the other maintainers will try our best to help with your contributions and our community is also a great one that can help you as well. 

One small tweak to the snippet and this will be good to go. Let me know once you've addressed it and I'll gladly merge it. With that in mind, we'll be looking forward to your next pull request with more Preact snippets.

Have a fantastic day!

Stay safe

File: docs/snippets/preact/button-story.js.mdx
Comment (fbredius) [reply docs/snippets/preact/button-story.js.mdx]: Thanks @jonniebigodes for your help! Have a great day as well.

**Comment by jonniebigodes** [comment]
Thanks for taking care of this so promptly, appreciate it. going to merge this!

**Pull request merged**
